services/frontend/src/scenes/ProjectDetails/ProjectDetails.scss

```diff
@@ -59,3 +59,34 @@
     }
 }
 
+#project-contributors{
```

Comment:
Two style things: there should be spaces between the brackets and the selectors, and there should be a newline between rules. I suggest making sure the rest of this file conforms to those rules.